.github/workflows/pr_label_check.yml

```
@@ -42,7 +42,7 @@ jobs:
       - get_label_groups
     steps:
       - name: Check aspect label
-        uses: docker://agilepathway/pull-request-label-checker:v1.6.56
+        uses: docker://agilepathway/pull-request-label-checker:v1.6.60
```

We can use `latest` here and stop having to update the workflow for each new release (till v2 comes around).

```suggestion
        uses: docker://agilepathway/pull-request-label-checker:latest
```

---

I looked at the latest releases here, and they mostly update dependencies, so probably a good idea to have the latest version